src/Razor/src/rzls/Program.cs

```diff
@@ -46,12 +46,11 @@ public static async Task MainAsync(string[] args)
                 }
             }
 
-            var server = await RazorLanguageServer.CreateAsync(
+            var server = RazorLanguageServerWrapper.Create(
                 Console.OpenStandardInput(),
                 Console.OpenStandardOutput(),
                 trace);
-            await server.InitializedAsync(CancellationToken.None);
```

Comment:
Out of curiosity, why do we not need to wait for initialization here anymore? Is it because the above call to `Create` handles it?

Reply:
Correct, InitializedAsync was an O# method that kicked off server startup (IIRC), and that's now taken care of by the Create method.